framerail/src/routes/[x+2d]/logout/+server.ts

```diff
@@ -0,0 +1,26 @@
+import { authLogout } from "$lib/server/auth/logout"
+
+export async function DELETE(event) {
+  let sessionToken = event.cookies.get("wikijump_token")
+
+  try {
+    let res = await authLogout(sessionToken)
+
+    event.cookies.delete("wikijump_token", {
+      path: "/",
+      httpOnly: true,
+      secure: true,
+      sameSite: "lax"
+    })
```

Comment:
Does this work? You mentioned it having issues.

Reply:
It works now, last time I tried I was debugging cookie issues and I think that may have affected something.